### ISSUE TITLE
Custom format idempotency

### ISSUE THREAD

**Issue by francoiseger** [devopsarr/ansible-collection-sonarr#67]
Custom formats with more than one condition do not seem to be idempotent, they register as changed even when no actual changes are made. This is likely due to a bug in the comparison between the state and desired custom format.

**Comment by Fuochi** [comment]
Hi @francoiseger,
Thanks for pointing this out! I'll have a look, and update the tests to check also this case

**Comment by Fuochi** [comment]
Hi @francoiseger,
I found the time to check this. However, I wasn't able to replicate the issue.
The following task with 2 conditions is idempotent:

```yaml
- name: test idempotency
  devopsarr.sonarr.sonarr_custom_format:
    include_custom_format_when_renaming: false
    name: "Language"
    specifications:
    - name: "language"
      implementation: "LanguageSpecification"
      negate: false
      required: true
      fields:
      - name: "value"
        value: 25
    - name: "size"
      implementation: "SizeSpecification"
      negate: false
      required: true
      fields:
      - name: "min"
        value: 10
      - name: "max"
        value: 100
    sonarr_api_key: "{{ sonarr_api_key }}"
    sonarr_url: "{{ sonarr_url }}"
```

Could you please share an example in which idempotency is broken?
Also, could you please share the sonarr version you are using?

**Comment by francoiseger** [comment]
This one is converted from trash custom formats:
```yaml
name: Multi-Audio
include_custom_format_when_renaming: false
specifications:
  - name: MULTi
    implementation: ReleaseTitleSpecification
    negate: false
    required: false
    fields:
      - name: value
        value: \b(MULTi(?![ .-]subs?))(\d|\b)
  - name: VO and VF
    implementation: ReleaseTitleSpecification
    negate: false
    required: false
    fields:
      - name: value
        value: ^(?=.*\b(VO(ST(FR)?)?)\b)(?=.*\b(VF[FIQ2]?)\b)
```

**Comment by Fuochi** [comment]
thanks, I'll have a look!

**Comment by Fuochi** [comment]
Hi @francoiseger,
This should be fixed in [v1.1.0](https://github.com/devopsarr/ansible-collection-sonarr/releases/tag/v1.1.0). Could you please check?

**Comment by francoiseger** [comment]
Yep, working as it should be now, thanks!
